weltyt.py

```diff
@@ -1,6 +1,115 @@
 import sqlite3
 
+def initializationDB(cursor):
 
+        with open("TYTDB.sql") as base:
+                for line in base:
+                        cursor.execute(line)
+
+lands_offert = {"humus": {"class": "humus", "id": 1, "growth_rate": 5, "price_ISL": 500, "plants": 0}, 
+                "humus_sag": {"class": "humus_sag", "id": 2, "growth_rate": 4, "price_ISL": 400, "plants": 0}, 
+                "brown_soil": {"class": "brown_soil", "id": 3, "growth_rate": 3, "price_ISL": 350, "plants": 0}, 
+                "lessive_soil": {"class": "lessive_soil", "id": 4, "growth_rate": 2, "price_ISL": 250, "plants": 0},
+                "clay_gravel": {"class": "clay_gravel", "id": 5, "growth_rate": 1, "price_ISL": 200, "plants": 0}}
+#INSERT INTO lands VALUES ("humus", 1, 5, 500, 0)
 print("Welcome to tyton")
 conn = sqlite3.connect("TYTDB.db")
 
+cur = conn.cursor()
+
+initializationDB(cur)        
+
+conn.commit()
+
+for row in cur.execute('SELECT * FROM resources'):
+        print(row)
+
+for rows in cur.execute('SELECT * FROM lands'):
+        print(rows)
+
+
+buying_plants = input("To grow tobbaco you have to buy some plants. If you want do it, write \"Plants\" please. ")
```

Comment:
I think on the beggining of the program we should just ask what do you want to do (buy or show resources)
If user selects 'buy' then we should ask what and based on their choice call proper `buying_plants` or `buying_lands` input.